src/core/ghosts.hpp

```diff
@@ -125,22 +125,21 @@ further details.
 /** \name Transfer data classes, for \ref GhostCommunication::type */
 /************************************************************/
 /*@{*/
-/// transfer \ref ParticleProperties
-#define GHOSTTRANS_PROPRTS 1
-/// transfer \ref ParticlePosition
-#define GHOSTTRANS_POSITION 2
-/// transfer \ref ParticleMomentum
-#define GHOSTTRANS_MOMENTUM 8
-/// transfer \ref ParticleForce
-#define GHOSTTRANS_FORCE 16
-
-/// resize the receiver particle arrays to the size of the senders
-#define GHOSTTRANS_PARTNUM 64
-
-#ifdef ENGINE
-/// transfer \ref ParticleParametersSwimming
-#define GHOSTTRANS_SWIMMING 128
-#endif
+enum : unsigned {
```

Comment:
type mismatch: change `GhostCommunication::type` to `unsigned int`
https://github.com/espressomd/espresso/blob/3923430b5cd560ba523292068d40dc92d04d7014/src/core/ghosts.hpp#L149-L151

Reply:
Those are unrelated

Reply:
Nevermind, I misread `GHOSTTRANS_*` values as `GHOST_*` values